src/elements/Label/Label.js

```diff
@@ -49,27 +46,29 @@ function Label(props) {
     className
   )
 
-  const _props = {
-    className: classes,
-    onClick: handleClick,
-    ...rest,
-  }
-
-  const _detail = detail || detailLink
-  const detailComponent = (detailLink || onClickDetail) && 'a' || detail && 'div'
+  const DetailComponent = (detailLink || onDetailClick) && 'a' || 'div'
```

Comment:
Would be nice to wrap the latter check in `()` for similar clarity to the initial expression.